gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageGrpcIntegrationTest.java

```diff
@@ -74,6 +75,22 @@ public void testCreateObject() throws IOException {
     assertObjectContent(rawStorage, objectToCreate, objectBytes);
   }
 
+  @Test
+  public void testReadWriteObjectWithTDEnabled() throws IOException {
```

Comment:
let's rename the test as `testOpen....` and move it below existing testOpen test